.eleventy.js

```diff
@@ -124,7 +124,7 @@ module.exports = function (config) {
     wrapper: 'div',
     wrapperClass: 'w-toc__list',
     ul: true,
-    flat: true,
+    flat: false,
```

Comment:
Is there a reason for this change?

Reply:
Yes, we want the Toc to be of a tree structure like in d.c.c and not flat.